docs/plots/backends.rst

```diff
@@ -0,0 +1,318 @@
+************************
+Plotting backend details
+************************
+
+.. warning::
+   The plotting backend API is currently re-designed and details might change in the near future.
+
+Plotting in Sherpa is done through *plotting backends*. A *backend* is an
+external package that performs the actual plotting function, i.e. it creates a
```

Comment:
you can add example of a backend in this sentence - 'for example matplotlib' package.